.buildkite/pipeline.yaml

```diff
@@ -46,12 +46,7 @@ steps:
         rm "$$tmp" &&
         captain --version
 
-      test_files=$$(
```

Comment:
💭 now setting `CAPTAIN_SUITE_ID` to ensure captain invocations use the same suite ID  is no longer necessary